src/lib/error.ts

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Based on https://stackoverflow.com/questions/57802057/eslint-configuring-no-unused-vars-for-typescript

YOu could remove all this lines, just try to update `typescript-eslint/recommended` to latest version

Reply:
Yeah, I just figured out this is fixed in the latest version

Reply:
@robertsLando one more question. It's now forcing a tabwidth of 2 spaces, is that correct?

Reply:
Dunno if it's correct or not I mean it's a matter of personal choice, you can choose the preferred indentation

Reply:
Yeah, I've set it back to 2. 
Kind of used to 4 in the python world ;-)